en/contributing/plugins.md

```diff
@@ -68,6 +68,58 @@ The [standard plugins](https://github.com/dronecore/DroneCore/tree/{{ book.githu
 how to write plugin code, including how to send and process MAVLink messages.
 
 
+### Plugin Base Class

```

Comment:
@julianoes This is new section. I added to provide a place for talking more about the code in future, if we so wish.

Reply:
That makes sense. There is not much now but there probably will be more in the future.